<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>productio_edits</class>
 <widget class="QFrame" name="productio_edits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>1010</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>504</width>
    <height>1010</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>504</width>
    <height>1010</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grpb">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16760000</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid #D8D4D3;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
     </property>
     <property name="title">
      <string>Edit Production</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <spacer name="vtsp_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>30</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="hzly_3">
        <item>
         <widget class="QRadioButton" name="rad_add">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Add Outline</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rad_edit">
          <property name="text">
           <string>Edit Outline</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <spacer name="vtsp_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lb_capture_method">
        <property name="text">
         <string>Capture Method:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="cmb_capture_method"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lb_capture_source">
        <property name="text">
         <string>Capture Source:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="cmb_capture_source"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lb_lifeycle_stage">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Lifecycle Stage:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="cmb_lifecycle_stage"/>
      </item>
      <item row="7" column="1">
       <spacer name="vtsp_1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="1">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <spacer name="vtsp_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="lb_town">
        <property name="text">
         <string>Town City:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QComboBox" name="cmb_town"/>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="lb_suburb">
        <property name="text">
         <string>Suburb Locality:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QComboBox" name="cmb_suburb"/>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="lb_ta">
        <property name="text">
         <string>Territorial Authority: </string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QComboBox" name="cmb_ta"/>
      </item>
      <item row="13" column="1">
       <spacer name="vtsp_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="15" column="0" colspan="2">
       <layout class="QHBoxLayout" name="hzly_4">
        <item>
         <widget class="QPushButton" name="btn_save">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_reset">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cancel">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>27</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Exit</string>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="16" column="0">
       <spacer name="vtsp_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="14" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_exit_edits">
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Exit Editing</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
     <zorder>lb_capture_method</zorder>
     <zorder>cmb_capture_method</zorder>
     <zorder>cmb_capture_source</zorder>
     <zorder>lb_capture_source</zorder>
     <zorder>lb_town</zorder>
     <zorder>cmb_town</zorder>
     <zorder>lb_suburb</zorder>
     <zorder>cmb_suburb</zorder>
     <zorder>lb_ta</zorder>
     <zorder>cmb_ta</zorder>
     <zorder>vtsp_6</zorder>
     <zorder>vtsp_7</zorder>
     <zorder>line</zorder>
     <zorder>vtsp_1</zorder>
     <zorder>vtsp_2</zorder>
     <zorder>cmb_lifecycle_stage</zorder>
     <zorder>lb_lifeycle_stage</zorder>
     <zorder>vtsp_3</zorder>
     <zorder>vtsp_4</zorder>
     <zorder>horizontalLayoutWidget</zorder>
    </widget>
   </item>
   <item>
    <spacer name="vtsp_8">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="vtsp_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
